README.md

```diff
@@ -319,6 +319,16 @@ custom:
       nodeModulesRelativeDir: '../../' # relative path to current working directory.
 ```
 
+> When using NPM 8, `peerDependencies` are automatically installed by default. In order to avoid adding all transitive dependencies to your package.json, we will use the `package-lock.json` when possible. If your project is included in a monorepo, you can specify the path to the `package-lock.json`:
```

Comment:
I think you don't need a citation block for that.
```suggestion
When using NPM 8, `peerDependencies` are automatically installed by default. In order to avoid adding all transitive dependencies to your `package.json`, we will use the `package-lock.json` when possible. If your project is included in a monorepo, you can specify the path to the `package-lock.json`:
```